saltboot-formula/_states/saltboot.py

```diff
@@ -965,7 +965,7 @@ def _get_image_for_part(images, part):
     image_id = part.get('image')
     image_version = part.get('image_version')
 
-    freeze = __salt__['pillar.get']('saltboot:freeze_image') or __salt__['pillar.get']('custom_info:saltboot_freeze_image')
+    freeze = __salt__['pillar.get']('saltboot:freeze_image') or __salt__['pillar.get']('custom_info:saltboot_freeze_image') in ['true', 'True']
```

Comment:
I'd assume that if somebody creates the  custom info entry, it means the non-default behavior is requested.
Therefore I would list values for False and interpret everything else as True